canvas.html

```diff
@@ -0,0 +1,240 @@
+<style>
+    body {
+        background-color: #242424;
+        color: #fff;
+    }
+
+    #imageDropdown{
+        height: 20rem;
+    }
+
+    #canvasContainer {
+        width: calc(800px * (16 / 9));
+        margin: 0 auto;
+    }
+
+    #canvas {
+        border: 1px solid black;
+        position: relative;
+        width: 100%;
+        height: 800px;
+    }
+
+    .dropdown-image {
+        width: 100px;
+        height: 80px;
+        vertical-align: middle;
+    }
+</style>
+
+<div class="container mt-3">
+    <div class="row mt-3">
+        <div class="col">
+            <div id="selectedImageInfo" class="mt-3"></div>
+        </div>
+    </div>
+    <div class="row mt-3">
+        <div class="col">
+            <button onclick="placeSticker()" class="btn btn-primary">Place Sticker</button>
+            <button onclick="saveCanvas()" class="btn btn-primary">Save Canvas</button> 
+        </div>
+    </div>
+    <div class="row mt-3 text-white fs-5">
+        <kbd>Backspace</kbd> = Delete Sticker | Mousewheel = Rotate Sticker | <kbd>Arrow Up</kbd> = 1 Layer Up | <kbd>Arrow Down</kbd> = 1 Layer Down
+    </div>
+
+</div>
+
+<div id="canvasContainer" class="mt-3">
+    <canvas id="canvas" width="1280" height="800"></canvas>
+</div>
+
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js"></script>
+<script>
```

Comment:
It would be great to put scripts in `.js` files

Reply:
Agreed on literally everything else, thanks for the review.